WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -232,6 +235,16 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 			LogCoinJoinSummary(registeredAliceClients, outputTxOuts, unsignedCoinJoin, roundState);
 
+			lock (SecondLargestDenomOutputLock)
```

Comment:
As I mentioned below the lock is not required.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/8757/commits/a2ce4d7836f2c3d8197def550b43636d04f520b2